packages/common/src/styles/theme.ts

```diff
@@ -175,13 +191,15 @@ const themeOptions = {
     },
   },
 };
-const theme = createTheme(themeOptions);
+export const createTheme = (themeOptions: ThemeOptions) => {
+  const theme = createMuiTheme(themeOptions);
 
-theme.shadows[1] =
-  '0 0.5px 2px 0 rgba(96, 97, 112, 0.16), 0 0 1px 0 rgba(40, 41, 61, 0.08)';
-theme.shadows[2] =
-  '0 4px 8px 0 rgba(96, 97, 112, 0.16), 0 0 2px 0 rgba(40, 41, 61, 0.04)';
-theme.shadows[3] =
-  '0 8px 16px 0 rgba(96, 97, 112, 0.16), 0 2px 4px 0 rgba(40, 41, 61, 0.04)';
-
-export default theme;
+  theme.shadows[1] =
+    '0 0.5px 2px 0 rgba(96, 97, 112, 0.16), 0 0 1px 0 rgba(40, 41, 61, 0.08)';
+  theme.shadows[2] =
+    '0 4px 8px 0 rgba(96, 97, 112, 0.16), 0 0 2px 0 rgba(40, 41, 61, 0.04)';
+  theme.shadows[3] =
+    '0 8px 16px 0 rgba(96, 97, 112, 0.16), 0 2px 4px 0 rgba(40, 41, 61, 0.04)';
+  return theme;
+};
+// export default theme;
```

Comment:
I prefer exporting the theme only, however it is only called in `AppTheme` and exporting the themeOptions allows options to be stored as text rather than a full theme object. Creating the theme outside of this file allows the custom shadows to be added too.